codalab/model/bundle_model.py

```diff
@@ -591,6 +591,61 @@ def batch_update_bundles(self, bundles, update, condition=None):
                 return success
         return True
 
+    def start_bundle(self, bundle, user_id, worker_id, hostname, start_time):
+        with self.engine.begin() as connection:
+            # Check that still assigned to this worker.
+            run_row = connection.execute(
+                cl_worker_run.select().where(cl_worker_run.c.run_uuid == bundle.uuid)).fetchone()
+            if not run_row or run_row.user_id != user_id or run_row.worker_id != worker_id:
+                return False
+
+            bundle_update = {
+                'state': State.RUNNING,
+                'metadata': {
+                    'remote': hostname,
+                    'started': start_time,
+                    'last_updated': start_time,
+                },
+            }
+            self.update_bundle(bundle, bundle_update, connection)
+
+        self.update_events_log(
+            user_id=bundle.owner_id,
+            user_name=None,  # Don't know
```

Comment:
Remove the user_name field from the events_log DB.  Because we can just look up the user_name in the user table (which didn't exist before).